### PR TITLE
[10.0] migrate all UNECE modules

### PR DESCRIPTION
**Pull request by alexis-via**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Can you please squash the history for removing meaningless commits?

**Comment by alexis-via** [comment]
Squashed.

**Review by pedrobaeza — APPROVED** [review]

File: account_payment_unece/README.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+This codification is used for example in the two main international standards for electronic invoicing:
+
+* `Cross Industry Invoice <http://tfig.unece.org/contents/cross-industry-invoice-cii.htm>`_ (CII),
+* `Universal Business Language <http://ubl.xml.org/>`_ (UBL)
+
+Configuration
+=============
+
+Go to the menu *Accounting > Configuration > Management > Payment Methods* and select the appropriate value for the *UNECE Code*.
```
Comment (tarteo) [inline account_payment_unece/README.rst]: I couldn't find this menu in Runbot.

**Review by tarteo — CHANGES_REQUESTED** [review]
Thanks for the migration! I was not able to test account_payment_unece (see comment), otherwise 👍

File: account_payment_unece/README.rst (hunk 1 above)
Comment (alexis-via) [reply account_payment_unece/README.rst]: True. You need the module account_payment_mode to see this menu entry, but we don't depend on this module.

**Review by tarteo — APPROVED** [review]

File: account_payment_unece/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+{
+    'name': 'Account Payment UNECE',
+    'version': '10.0.1.0.0',
+    'category': 'Accounting & Finance',
+    'license': 'AGPL-3',
+    'summary': 'UNECE nomenclature for the payment methods',
+    'author': 'Akretion,Odoo Community Association (OCA)',
+    'website': 'http://www.akretion.com',
+    'depends': ['account_payment_mode', 'base_unece'],
```
Comment (pedrobaeza) [inline account_payment_unece/__manifest__.py:14]: @alexis-via the dependency to `account_payment_mode` is here, so the comment is not correct.

**Review by astirpe — APPROVED** [review]

**Comment by alexis-via** [comment]
Someone with commit rights to merge this ?

**Comment by pedrobaeza** [comment]
Can I squash your commits?

**Comment by alexis-via** [comment]
I squashed the 2 commits I made before my last squash !

**Pull request merged**